examples/AddressBook/AddressBookPlugin.cpp

```diff
@@ -43,11 +43,14 @@ void AddressBookPlugin::registerTypes(const char *uri)
 {
     // Register the generated types
     Module::registerQmlTypes(uri);
-    Module::registerUncreatableQmlTypes(uri);
+
+    // Register uncreatable type as "AddressBook" (default name)
+    facelift::registerUncreatableQmlComponent<AddressBook>(uri);
 
     // We are registering the model types here, which can be used by the UI code.
     // The decision to register a dummy, QML, or C++ implementation should be taken here
-    facelift::registerQmlComponent<AddressBookCppWithProperties>(uri, "AddressBook");
+    facelift::registerQmlComponent<AddressBookCppWithProperties>(uri, "AddressBookImpl");
```

Comment:
Didn't we agree not to register plugin name ending with *Impl ? or correct me if i am wrong